packages/client-utils/src/vstorage-kit.js

```diff
@@ -96,54 +97,90 @@ export const makeAgoricNames = async (ctx, vstorage) => {
 };
 
 /**
- * @param {{ fetch: typeof window.fetch }} io
- * @param {MinimalNetworkConfig} networkConfig
+ * @param {object} config
+ * @param {VStorage} config.vstorage
+ * @param {MinimalNetworkConfig} config.networkConfig
+ * @param {Omit<Marshal<string>, 'serialize' | 'unserialize'>} [config.marshaller]
```

Comment:
Would it be better to describe what methods we want as opposed to the ones we don't want?

```suggestion
 * @param {Pick<Marshal<string>, 'fromCapData' | 'toCapData'>} [config.marshaller]
```